### ISSUE TITLE
Request: Please Allow Removing Google Search

### ISSUE THREAD

**Issue by 0pLuS0** [amirzaidi/Launcher3#558]
Hello,

I don't use any Gapps on my phone all custom rom, so I have no need for a Search, and even if some people do, many like the option/choice of removing or changing placement.

Please make an option to remove/disable this...

Thank you

**Comment by vogtmh** [comment]
Yes please! In my case, it only wastes screen space. An option to hide it would be nice :-)

**Comment by 0pLuS0** [comment]
I forgot to mention, one other very important feature would be the ability edit/change the labels, renaming them, or changing their font size so we can read the entire label.

It's really sad to see label names cut off... 

Thanks

**Comment by Rjdiseno** [comment]
Hi,
Yes, please can you make an option to remove or hide the google search bar on the home screen. It would be very nice, thank you.

**Comment by Pandapip1** [comment]
If this is still being developed, I am also +1 to this idea.
